src/silverlabnwb/timings.py

```diff
@@ -0,0 +1,101 @@
+import abc
+
+import numpy as np
+import pandas as pd
+
+
+class LabViewTimings(metaclass=abc.ABCMeta):
+
+    def __init__(self, relative_times_path, roi_path, dwell_time):
+        self.dwell_time = dwell_time
+        self._read_relative_times_file(relative_times_path)
+        self._read_roi_data(roi_path)
+
+    @abc.abstractmethod
+    def _read_relative_times_file(self, file_path):
+        pass
+
+    def _read_roi_data(self, roi_path):
+        self.roi_data = pd.read_csv(roi_path, sep='\t', dtype=np.float64)
+        self.n_rois = len(self.roi_data['ROI index'])
+        # in the future, we can access more shape parameters here too, for variable size ROIs in those cases,
+        # we will need to access each individual row here separately. for now, take first row and all ROIs are same
+        # size and orientation. Also, we may need to store the angle somewhere in the future (to reconstruct the ROI
+        # in 3D, in which case the ==0 assertion may become imprecise and we would need a tolerance to compare with a
+        # double value.
```

Comment:
Is this something we think will definitely be needed or only possibly? I don't remember!

Reply:
I thought so, but now not sure... we should ask again at the next catch-up?